### ISSUE TITLE
Seahorses wont stop spawning

### ISSUE THREAD

**Issue by OQuofer** [c0d3break3r/Omni#40]
The seahorses constantly spawn to the point where there are hundreds of them and lag my game. I tried to lower the spawn rate but that has no affect and when I set it to 0 my game crashes because it tries to spawn them.

**Comment by saruteku** [comment]
More information. I set up my base on a tropical island from Biomes o Plenty next to a coral reef. They do not seem affected by the config to how often they spawn at all. I set up a command block to transport them to the void but at least one seems to be spawned every tick, and I also found they spawn uncontrolled the higher I get, with my command block teleporting up to 500 per tick at cloud level. It works fine if I am not near the coral reef, but they are problematic as it turned my island nearly unplayable if I do not periodically kill them. It has been the only mob I have dealt with odd spawn habits so far.

**Comment by TelepathicGrunt** [comment]
The reason why the seahorses will not stop spawning is because they are registered as a Creature but added to the biome as a Water_ambient mob. The mismatch in the entity classification means the seahorse will spawn during the water ambient phase but only counts towards the creatures mob cap instead of the water ambient phase. And so, the seahorses will spawn non-stop like mad. You can try spawning a bunch of water ambient mobs such as fish till you force the mob cap to activate and stop the seahorses

https://github.com/c0d3break3r/Omni/blob/eb7dcef76b27b7b3f1685acc1e0c6c7e6daff553/src/main/java/pugz/omni/core/module/ParadiseModule.java#L167

https://github.com/c0d3break3r/Omni/blob/eb7dcef76b27b7b3f1685acc1e0c6c7e6daff553/src/main/java/pugz/amni/core/module/ParadiseModule.java#L103

**Comment by Zygus42** [comment]
Yep - same problem.  Killing our server.  Anyone know of the syntax for a server command to just kill all the seahorses ? These are some pictures:
![2021-02-19_20 34 20](https://user-images.githubusercontent.com/20757445/108506311-88a3fd00-72f3-11eb-9106-9ec4877f1a74.png)
![2021-02-19_20 38 29](https://user-images.githubusercontent.com/20757445/108506317-8b065700-72f3-11eb-8772-216c5e9fd6cf.png)

**Comment by TelepathicGrunt** [comment]
@Zygus42 `/kill @e[type=omni:seahorse]` should work. Maybe a repeating command block with that could work but make sure there's a delay so the command doesn't lag too much

**Comment by Zygus42** [comment]
Thanks for that - logged on and killed 4158 entities.

I've set the spawning on  the horses to almost nothing, so shouldn't see this much buildup again once i get rid of the existing ones,.

**Comment by Zygus42** [comment]
No such luck - set the spawn chance from 0.75 to 0.01 and an hour later had to kill another 1000.  Will try setting it to 0 i guess.

**Comment by Zygus42** [comment]
OK - set it to 0, and still getting tons of seahorse spawning and not despawning.
